src/test/scala/dibl/Demo4Java.java

```diff
@@ -29,44 +32,52 @@ public static void main(String[] args) throws Throwable {
-            + "&shiftColsSW=0&shiftRowsSW=4&shiftColsSE=4&shiftRowsSE=4",
+    String[] urlQueries = { //
+        "patchWidth=5&patchHeight=5"
+            + "&tile=5-&tileStitch=ct&"
+            + "&shiftColsSW=-1&shiftRowsSW=1&shiftColsSE=1&shiftRowsSE=1",
         "patchWidth=11&patchHeight=12" //
-            + "&tile=B-C-,---5,C-B-,-5--&tileStitch=ctct"
-            + "&shiftColsSW=0&shiftRowsSW=4&shiftColsSE=4&shiftRowsSE=4"
+            + "&tile=B-C-,---5,C-B-,-5--&tileStitch=ct"
+            + "&shiftColsSW=0&shiftRowsSW=4&shiftColsSE=4&shiftRowsSE=4",
         // TODO for more examples: see the demo section of the tiles page
+        //  patch size must be at least 4 rows/columns larger than the tile
+        //  tiling with gaps might not work
     };
-    for (int i = 0; i <= urlQuerys.length - 1; i++) {
-      generateSetOfDiagrams(urlQuerys[i], i);
+    for (int i = 0; i <= urlQueries.length - 1; i++) {
+      generateSetOfDiagrams(urlQueries[i], i);
     }
   }
 
   private static void generateSetOfDiagrams(String urlQuery, int i) throws IOException {
-    Diagram pairs = NewPairDiagram.create(Config.create(urlQuery));
-    Diagram nudgedPairs = generateSingleDiagram(i + "-pairs", "1px", pairs);
-    Diagram threads = ThreadDiagram.create(nudgedPairs);
-    Diagram nudgedThreads = generateSingleDiagram(i + "-threads", "2px", threads);
-    Diagram drostePairs = PairDiagram.create("ctct", nudgedThreads);
-    Diagram nudgedDroste = generateSingleDiagram(i + "-droste-pairs", "1px", drostePairs);
-    Diagram drosteThreads = ThreadDiagram.create(nudgedDroste);
-    generateSingleDiagram(i + "-droste-threads", "2px", drosteThreads);
+    Config config = Config.create(urlQuery);
+    Diagram pairs = NewPairDiagram.create(config);
+    generateDiagram(i + "-pairs", "1px", pairs, config, 1);
+    Diagram threads = ThreadDiagram.create(pairs);
+    generateDiagram(i + "-threads", "2px", threads, config, 2);
+    Diagram drostePairs = PairDiagram.create("ctct", threads);
+    generateDiagram(i + "-droste-pairs", "1px", drostePairs, config, 2);
+    Diagram drosteThreads = ThreadDiagram.create(drostePairs);
+    generateDiagram(i + "-droste-threads", "2px", drosteThreads, config, 4);
   }
 
-  private static Diagram generateSingleDiagram(String fileName, String strokeWidth, Diagram diagram)
-      throws IOException {
-    // TODO make a copy of the node sequence using diagram.node(i).withLocation(x,y)
-    Seq<NodeProps> nudgedPairNodes = diagram.nodes();
+  private static void generateDiagram(
+      String fileName,
+      String strokeWidth,
+      Diagram diagram,
+      Config config,
+      Integer scale
+  ) throws IOException {
+    System.out.println("-------------- "+fileName);
 
-    Diagram nudgedDiagram = new Diagram(nudgedPairNodes, diagram.links());
+    NodeProps[][] links = config.centerTile(diagram, scale);
+    // TODO calculate deltas from links and Config.shiftXxxx
```

Comment:
@veronika 
Your IDE should give you easy access to the scala-doc of `centerTile`. The method will log the result on standard-out.
Please give me feed back if this does not give you the data you need or you spot a bug.